qemu_v8.mk

```diff
@@ -470,6 +471,24 @@ else
 QEMU_MTE	= off
 endif
 
+QEMU_ARGS := \
+	-nographic \
+	-smp $(QEMU_SMP) \
+	-s -S -machine virt,acpi=off,secure=on,mte=$(QEMU_MTE),gic-version=$(QEMU_GIC_VERSION),virtualization=$(QEMU_VIRT) \
+	-cpu $(QEMU_CPU) \
+	-d unimp -semihosting-config enable=on,target=native \
+	-m $(QEMU_MEM) \
+	-bios bl1.bin		\
+	-initrd rootfs.cpio.gz \
+	-kernel Image \
+	-append 'console=ttyAMA0,38400 keep_bootcon root=/dev/vda2 $(QEMU_KERNEL_BOOTARGS)' \
+	$(QEMU_XEN) \
+	$(QEMU_EXTRA_ARGS)
+	
+ifeq ($(QEMU_USE_CUSTOM_DTB),y)
+QEMU_DTB_ARG	?= -dtb $(QEMU_DTB_PATH)
+endif
```

Comment:
can replace this change with:
```makefile
ifeq ($(QEMU_USE_CUSTOM_DTB),y)
QEMU_EXTRA_ARGS += -dtb $(QEMU_DTB_PATH)
endif

Reply:
When QEMU receives both `dtb` and `dumpdtb` arguments it wants to boot with that DTB, so it won't dump anything, but `QEMU_EXTRA_ARGS` needs to be passed to QEMU when dumping DTB as it has an effect on the DTB.